Source/WebCore/html/HTMLTagNames.in

```diff
@@ -45,7 +45,7 @@ div
 dl interfaceName=HTMLDListElement
 dt interfaceName=HTMLElement
 em interfaceName=HTMLElement
-embed
+embed interfaceName=HTMLEmbedElement, settingsConditional=embedElementEnabled
```

Comment:
Do we really need the `interfaceName=HTMLEmbedElement`? Seems like this should be the default.